incubator/binding-asyncapi.spec/src/main/scripts/io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.schema.patch.json

```diff
@@ -82,28 +94,7 @@
                                 {
                                     "channels":
                                     {
-                                        "title": "Channels",
-                                        "type": "object",
-                                        "properties":
-                                        {
-                                            "sessions":
-                                            {
-                                                "title": "Kafka Sessions Channel",
-                                                "type": "string"
-                                            },
-                                            "messages":
-                                            {
-                                                "title": "Kafka Messages Channel",
-                                                "type": "string"
-                                            },
-                                            "retained":
-                                            {
-                                                "title": "Kafka Retained Channel",
-                                                "type": "string"
-                                            },
-                                            "additionalProperties": false
-                                        },
-                                        "additionalProperties": false
+                                        "$ref": "#/$defs/options/binding/mqtt-kafka/topics"
```

Comment:
I'm not sure about this one, since we're deliberately using `channels` asyncapi concept here, whereas mqtt-kafka deals with kafka-specific `topics` instead, so unifying this config causes asyncapi to switch to `topics` instead.